ci/terraform/account-management/variables.tf

```diff
@@ -224,6 +224,12 @@ variable "support_email_check_enabled" {
   description = "Feature flag which toggles the Experian email check on and off"
 }
 
+variable "txma_audit_encoded_enabled" {
```

Comment:
Is there a requirement to feature flag this? (I didn't think so but might have missed it)